frontend/views/containers/payments/RecordPayment.vue

```diff
@@ -157,19 +166,26 @@ export default {
         })
       }
 
-      for (const payment of this.ourPayments.todo) {
-        todoPayments.push({
-          hash: payment.hash,
-          username: payment.to,
-          displayName: this.userDisplayName(payment.to),
-          amount: payment.amount,
-          total: payment.total,
-          partial: payment.partial,
-          isLate: false,
-          date: payment.dueIn
+      const dueOn = this.lastDayOfThisMonth
+      groupIncomeDistribution({
+        getters: {
+          groupProfiles: this.groupProfiles,
+          groupSettings: this.groupSettings,
+          currentGroupState: this.currentGroupState,
+          monthlyPayments: this.groupMonthlyPayments
+        },
+        monthstamp: currentMonthstamp,
+        adjusted: true
+      }).filter((i) => i.to === this.ourUsername)
```

Comment:
OK, I have no idea why this was modified to be this... `this.ourPayments.todo` is supposed to be used here, and that in turn is supposed to call `groupIncomeDistribution` (down the line). So putting it here doesn't make sense.

Reply:
Thanks for explaining this section of code.

I believe I modified it while attempting to make the cypress test work.

I think it snuck past, like the `package-lock.json`. I will change it back in the next commit.